### PR TITLE
Chocolate Classification and Prediction

### PR DESCRIPTION
**Pull request by PranjalRkgp**
# Pull Request for ML-Crate 💡

## Issue Title: Chocolate Classification and Prediction

- **Info about the related issue (Aim of the project)** : Classify the varieties of the chocolates and predict them accordingly.
- **Name:** Pranjal Rathore
- **GitHub ID:** PranjalRkgp
- **Idenitfy yourself:** Contributing as a participant in KWOC(Kharagpur Winter Of Code)

Closes: #319 

### Describe the add-ons or changes you've made 📃

Folders added: Dataset, Images, Model, requirements(.txt file)
 -they include a kaggle dataset link, 3 outsourced test images, a .ipynb file of model.

## Type of change ☑️

What sort of change have you made:

- [x] Added new files and folders with the aim to solve this new issue, details have been mentioned above
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested : on Google colab
Describe how have you verified the changes made: with outsourced test images and  plotting the accuracy of prediction with the score of accuracy also being mentioned and considered.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Please add the `README.md` file, follow the template for creating the `README.md` file.
@PranjalRkgp

**Comment by PranjalRkgp** [comment]
@abhisheks008  Please have a look now on the changes, I added the README.md file.

File: Chocolate Classification and Prediction/Model/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## LIBRARIES/FRAMEWORK NEEDED
+
+- NumPy
+- Matplotlib
+- PIL
+- TensorFlow
+
+## PLOTS
+
+!(https://drive.google.com/file/d/1xY8Fw7ifkY-N1tGa6axgQ0eJxaVSM0Ix/view?usp=sharing)
```
Comment (abhisheks008) [inline Chocolate Classification and Prediction/Model/README.md:30]: You can directly put the visualization images in the README.md file.
For example, `[](Image url)` You can fetch the URL from the Images folder where you have uploaded the images for this project.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Chocolate Classification and Prediction/Model/README.md (hunk 1 above)
Comment (PranjalRkgp) [reply Chocolate Classification and Prediction/Model/README.md]: i tried doing it in the latest attempts...but it not image is not showing directly in the README.md file.

Comment (abhisheks008) [reply Chocolate Classification and Prediction/Model/README.md]: Try to use this tag,
``

Comment (abhisheks008) [reply Chocolate Classification and Prediction/Model/README.md]: Or you can use this also,
``

My bad, I missed the `!` sign.

File: Chocolate Classification and Prediction/Model/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## LIBRARIES/FRAMEWORK NEEDED
+
+- NumPy
+- Matplotlib
+- PIL
+- TensorFlow
+
+## PLOTS
+![](Chocolate Classification and Prediction/Images/plots.png)
```
Comment (abhisheks008) [inline Chocolate Classification and Prediction/Model/README.md:29]: insert the URL

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Comment by PranjalRkgp** [comment]
every way ,i tried , its not showing...

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

File: Chocolate Classification and Prediction/Model/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## LIBRARIES/FRAMEWORK NEEDED
+
+- NumPy
+- Matplotlib
+- PIL
+- TensorFlow
+
+## PLOTS
+![](https://drive.google.com/file/d/1xY8Fw7ifkY-N1tGa6axgQ0eJxaVSM0Ix/view?usp=sharing)
```
[the remark this answers is not in the document]
Comment (abhisheks008) [reply Chocolate Classification and Prediction/Model/README.md]: Similarly add all the images from your Images folder one after another.

**Comment by PranjalRkgp** [comment]
nope...its not coming ...i  have tried them as well
..wouldn't it be fine with just giving a link to image ...if its not
showing in any of the ways ?

**Comment by abhisheks008** [comment]
It worked, check from your end. It's showing perfect in my end.

**Comment by abhisheks008** [comment]
image
Look at this!

**Comment by PranjalRkgp** [comment]
okay...great...thanks !
I added test images also...
please have a look and approve my PR, if it works well.
thanks a lot !

**Comment by PranjalRkgp** [comment]
Is it okay? , all fine with PR ?

**Review by abhisheks008 — APPROVED** [review]
Looks good to me!
Your PR is approved and ready to be merged.
@PranjalRkgp

**Comment by PranjalRkgp** [comment]
Thanks a lot. 🙏

**Pull request merged**